### ISSUE TITLE
Are you using Spring Boot Database Admin?

### ISSUE THREAD

**Issue by aileftech** [aileftech/spring-boot-database-admin#17]
If you're using this project, I would love to know what are you using it for and how it's performing. This helps me a lot in understanding what features to prioritize, what's working and what not, etc... Thanks for any feedback!

**Comment by fabienfleureau** [comment]
Hello,
Thanks for you project.
I use it an hidden admin page, mostly as if it is a direct connection to database. 
It would be nice to able to run custom queries.

**Comment by Albertevc** [comment]
Hello guys, awesome project!
mostly i use the sql console, it blowed my mind.

But currently i had to deactivate the scan for some of my entities, because some of them have INHERITANCE.
It would be nice to have support for those.

Many thanks!

**Comment by ketanmane2882** [comment]
how to connect the springBoot crud Project  to the snapadmin

**Comment by haeser** [comment]
Hi @aileftech pretty cool project. Was there any particular reason for why you selected GPLv3 as a license?
From experience I can tell that adoption, especially using it in professional settings at organisations will be hindered by it since GPLv3 gets always flagged by various tools.
Apache or MIT would be more permissive and people still contribute back.

**Comment by aileftech** [comment]
Hi @haeser, thanks! When I started the project, I wanted to keep a door open in case I wanted to release a commercially-licensed "pro" version later. Since I am not really interested in doing that anymore I am considering switching the license to MIT, as I'm aware it hinders adoption.

Out of curiosity, were you actually planning to use it in a professional setting or you were talking in general?

**Comment by haeser** [comment]
> Hi @haeser, thanks! When I started the project, I wanted to keep a door open in case I wanted to release a commercially-licensed "pro" version later. Since I am not really interested in doing that anymore I am considering switching the license to MIT, as I'm aware it hinders adoption.
> 
> Out of curiosity, were you actually planning to use it in a professional setting or you were talking in general?

More like in general. Having a background in Rails, simple commodities like active admin are simply missing in spring boot and that was bothering me (in the past) for a variety of projects. It only happened randomly that I stumbled over this project recently.
The comment about the license was more like sharing my experience. It was never an issue for any of the companies I worked for, using and especially contributing back to projects that have less strict licenses whereas GPL was always a red flag.

**Comment by aileftech** [comment]
I understand, thanks for the detailed answer! Given the circumstances, I think I'll most likely change  the license some time soon. I haven't touched the code for a while so I'll sneak in a couple of bug fixes that are still to be released and probably do everything altogether with next release.

**Comment by rosarioconti** [comment]
Ciao Guys,
 Are you interested to compile the project to Kotlin in parallel ? 

 I am also running problems with :
 java.lang.UnsupportedClassVersionError: tech/ailef/snapadmin/external/SnapAdminAutoConfiguration has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 59.0

Since I'm on 18

**Comment by aileftech** [comment]
Hi there,
I've just released a new `0.2.1` version which includes some small bugfixes but most importantly the switch to MIT license, which is now live on the repo too. 

I hadn't performed the process in a few months so I hope everything is alright. The artifact should be available on Maven soon.

**Comment by aileftech** [comment]
Alright, the package is on Maven but I forgot the change the version number in the UI so:

> NOTE: Version number has not been updated correctly in the Maven released package. The UI on the bottom left will still show `0.2.0` as version number for this release instead of `0.2.1`.

**Comment by parsaM110** [comment]
I recently came across Snap Admin, and I must say it's a very interesting project! Coming from a Django background, I always wondered why there wasn't a robust ecosystem like Django Admin in the Spring world. After discovering that Spring Roo was no longer supported by VMware, I found your project and was genuinely impressed.

Keep up the great work!
